cypress/integration/console/devices/create-manual.spec.js

```diff
@@ -541,12 +523,14 @@ describe.skip('End device manual create', () => {
         cy.findByLabelText('Define multicast group (ABP & Multicast)').check()
         cy.findByLabelText('LoRaWAN class for multicast downlinks').selectOption('class-b')
```

Comment:
Please add a check here that `cy.findByLabelText('Network defaults').should('not.be.checked').and('have.attr', 'disabled')`